src/test/java/org/apache/commons/imaging/formats/jpeg/segments/JfifSegmentTest.java

```diff
@@ -0,0 +1,35 @@
+package org.apache.commons.imaging.formats.jpeg.segments;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+/**
+ * Unit tests for class {@link JfifSegment}.
+ *
+ * @date 13.07.2017
+ * @see JfifSegment
+ *
+ **/
+public class JfifSegmentTest{
+
+
+  @Test
+  public void testCreatesJfifSegment() {
+
+      byte[] byteArray = new byte[25];
+      JfifSegment jfifSegment = null;
+
+      try {
+        jfifSegment = new JfifSegment((-2275), byteArray);
+        fail("Expecting exception: Exception");
+      } catch(Throwable e) {
```

Comment:
Had to do with the fact that at the beginning the test class resided in a different hierarchical folder structure than the original class ...
Corrected.